Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/NetworkSearchController.java

```diff
@@ -96,4 +106,30 @@ public String getSearch(@PathVariable("projectUnixName") String projectUnixName,
         return new ResponseEntity<String>(defaultJsonErrorMsg, HttpStatus.INTERNAL_SERVER_ERROR);
     }
 
+    @RequestMapping(value = "sites/{projectUnixName}/networks/search", method = RequestMethod.GET)
+    public String getSearchTransformedNetwork(@PathVariable("projectUnixName") String projectUnixName,
+                                              @RequestParam("conceptId") String conceptId,
+                                              Model model)
+        throws JAXBException, QuadrigaStorageException {
```

Comment:
where is the JAXBException thrown? You should handle it and not just pass it on.